packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -15,11 +16,33 @@ export default class Sayt extends LitElement {
    * Determines the visibility of the `sayt` component.
    */
   @property({ type: Boolean, reflect: true }) visible = false;
+  /**
+   * Stores the ID of the relevant search element.
+   */
+  @property({ type: String, reflect: true }) searchbox = '';
```

Comment:
Precedent in HTML says otherwise. See the `form` attribute for the various form elements, `for` for `<label>`, and loosely `usemap` for `<img>` elements.